jflex/src/main/java/jflex/Emitter.java

```diff
@@ -921,10 +921,10 @@ private void emitLexFunctHeader() {
       println("        zzCh = Character.codePointAt(zzBufferL, zzCurrentPosL, zzMarkedPosL);");
       println("        zzCharCount = Character.charCount(zzCh);");
       println("        switch (zzCh) {");
-      println("        case '\\u000B':");
-      println("        case '\\u000C':");
-      println("        case '\\u0085':");
-      println("        case '\\u2028':");
+      println("        case '\\u000B':  // fall though");
+      println("        case '\\u000C':  // fall though");
+      println("        case '\\u0085':  // fall though");
+      println("        case '\\u2028':  // fall though");
```

Comment:
@regisd - I know this was merged a long time ago, but "through" is consistently misspelled, FYI.

Reply:
Indeed, thanks!